homeassistant/components/anova_sous_vide/config_flow.py

```diff
@@ -0,0 +1,34 @@
+"""Config flow for Anova."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+
+class AnovaConfligFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Sets up a config flow for Anova."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Handle a flow initiated by the user."""
+        if user_input is not None:
+            entries = self._async_current_entries()
+            if any(x.data["device_id"] == user_input["device_id"] for x in entries):
+                return self.async_abort(reason="already_configured")
+
```

Comment:
`_abort_if_unique_id_configured` is doing the same thing here

Reply:
After making this change, `test_flow_user_already_configured` fails. Any insight you can give?